examples/historic_async.py

```diff
@@ -0,0 +1,126 @@
+import sys
+import asyncio
+import alpaca_trade_api as tradeapi
+from alpaca_trade_api.rest import TimeFrame, URL
+from alpaca_trade_api import rest_async
+from alpaca_trade_api.rest_async import gather_with_concurrency
+from alpaca_trade_api import AsyncRest
+import pandas as pd
+
+NY = 'America/New_York'
```

Comment:
unused

Reply:
it's used in `async def main(symbols):` but it was defined twice. removed one.